### PR TITLE
fix deepar docstring

### PR DESCRIPTION
**Pull request by lostella**
*Description of changes:* The current docstring for `DeepAREstimator` is out of sync, this PR puts it back in sync with the code.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/gluonts/model/deepar/_estimator.py [hunk 1]
```diff
@@ -72,8 +72,15 @@ class DeepAREstimator(GluonEstimator):
...
         Dropout regularization parameter (default: 0.1)
+    use_feat_dynamic_real
+        Whether to use the ``feat_dynamic_real`` field from the data
+        (default: False)
+    use_feat_static_cat
+        Whether to use the ``feat_static_cat`` field from the data
+        (default: False)
     cardinality
-        Number of values of the each categorical feature (default: [1])
+        Number of values of the each categorical feature.
```
Comment (jaheba) [inline src/gluonts/model/deepar/_estimator.py]: The `the` looks to much in this line

Comment (lostella) [reply src/gluonts/model/deepar/_estimator.py]: That’s a typo indeed

Comment (lostella) [reply src/gluonts/model/deepar/_estimator.py]:
```suggestion
        Number of values of each categorical feature.
```

**Comment by szha** [comment]
Job PR-195/2 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-195/2/index.html

**Pull request merged**